### PR TITLE
Added documentation and images on the new Cloud tabs

### PR DESCRIPTION
**Pull request by suyogsainkar**
This pull request includes content from a pending merge request in gitlab which was tech/peer reviewed but unmerged there.

### CODE REVIEW

**Comment by adahms** [comment]
@suyogsainkar 

Have the required changes been made in the index.adoc file in the miq directory as well?

**Comment by suyogsainkar** [comment]
@adahms 
Thanks - I've made the changes in the index.adoc file in the miq directory now.

File: doc-Managing_Infrastructure_and_Inventory/topics/Tagging_Object_Stores.adoc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+= Tagging Object Stores 
+
+Apply tags to object stores to categorize them together at the same time. You can create tags using the instructions in the https://access.redhat.com/documentation/en/red-hat-cloudforms/4.1/general-configuration/general-configuration[General Configuration] guide.
```
Comment (dayleparker) [inline doc-Managing_Infrastructure_and_Inventory/topics/Tagging_Object_Stores.adoc]: Should there be an ifdef condition added in for this link?

File: doc-Managing_Infrastructure_and_Inventory/topics/Tagging_Orchestration_Stacks.adoc [hunk 2]
```diff
@@ -0,0 +1,11 @@
+= Tagging Orchestration Stacks
+
+Apply tags to orchestration stacks to categorize them together at the same time. You can create tags using the instructions in the https://access.redhat.com/documentation/en/red-hat-cloudforms/4.1/general-configuration/general-configuration[General Configuration] guide.
```
Comment (dayleparker) [inline doc-Managing_Infrastructure_and_Inventory/topics/Tagging_Orchestration_Stacks.adoc]: Should there be an ifdef condition added in for this link?

Comment (adahms) [inline doc-Managing_Infrastructure_and_Inventory/topics/Tagging_Orchestration_Stacks.adoc]: @dayleparker @suyogsainkar 

Hmm, do we need this reference? It doesn't really point to a specific section, and I think we should consider even adding the procedure for creating tags to the same guide or collecting all relevant sections in the same place at some stage, for example.

What do you think?

Comment (suyogsainkar) [inline doc-Managing_Infrastructure_and_Inventory/topics/Tagging_Orchestration_Stacks.adoc]: @adahms Yeah, good point. And certainly worth collecting all relevant sections in a workflow in the same place where possible at some point.

**Comment by dayleparker** [comment]
Looks good Suyog!

**Pull request merged**